Kudu.Contracts/Settings/DeploymentSettingsExtension.cs

```diff
@@ -218,5 +218,18 @@ public static bool TouchWebConfigAfterDeployment(this IDeploymentSettingsManager
         {
             return settings.GetValue(SettingsKeys.TouchWebConfigAfterDeployment) != "0";
         }
+
+        public static bool IsDockerCiEnabled(this IDeploymentSettingsManager settings)
+        {
+            string value = settings.GetValue(SettingsKeys.DockerCiEnabled);
+            return StringUtils.IsTrueLike(value);
+        }
+
+        public static bool RestartAppContainerOnGitDeploy(this IDeploymentSettingsManager settings)
+        {
+            // Default is true
+            string value = settings.GetValue(SettingsKeys.LinuxRestartAppContainerAfterDeployment) ?? "true";
+            return StringUtils.IsTrueLike(value);
```

Comment:
nit: slightly ugly to hard code "true" and then parse it. `return value == null || StringUtils.IsTrueLike(value);` is a bit cleaner.

Reply:
Ah thank you, I thought this smelled funny too. I think I wrote this when the coffee machine was broken.